lib/adapters/REST/endpoints/entry.ts

```diff
@@ -30,6 +30,22 @@ export const get: RestEndpoint<'Entry', 'get'> = <T extends KeyValueMap = KeyVal
   )
 }
 
+export const getPublishedEntries:RestEndpoint<'Entry', 'getPublishedEntries'> = <T extends KeyValueMap = KeyValueMap>(
```

Comment:
here we can keep the old name - `getPublished` because users will call it as `entry.getPublished` and it will already be clear that they are requesting entries

with the current name they would call it as `entry.getPublishedEntries` so the word `entry` appears twice and it is unnecessary